packages/commonwealth/client/styles/components/component_kit/cw_component_showcase.scss

```diff
@@ -200,7 +200,7 @@ $width: 360px;
       gap: 8px;
       margin-bottom: 15px;
 
-      > .Text {
+      >.Text {
```

Comment:
again, this formatting does not seem to be right

Reply:
Updated my editor so this should stop happening. Thanks!